README.md

```diff
@@ -47,13 +47,51 @@ You can also set `-Zmiri-start-fn` to make Miri start evaluation with the
 
 ## Running Miri on your own project('s test suite)
 
-Install Miri as a cargo subcommand with `cargo install --all-features`, and install
-a full libstd as described above.
+Install Miri as a cargo subcommand with `cargo install --all-features --path .`.
```

Comment:
This and everything above it in the README still lacks a `+nightly`. The instructions as they are now just don't work.